components/SpotInfo/index.js

```diff
@@ -109,10 +230,36 @@ export default function SpotInfo({ spotId }) {
           <EntryList>
```

Comment:
Your ``<EntryList>`` is a ``ul``. That's fine. But the direct children are ``<div>``s! They have to be ``<li>``s.
Your ``<li>`` is two areas below.